src/icalendar/timezone/zoneinfo.py

```diff
@@ -0,0 +1,133 @@
+"""Use zoneinfo timezones"""
+from __future__ import annotations
+try:
+    import zoneinfo
+except:
+    from backports import zoneinfo
+from icalendar import prop
+from dateutil.rrule import rrule, rruleset
+from datetime import datetime, tzinfo
+from typing import Optional
+from .provider import TZProvider
+from .. import cal
+from io import StringIO
+from dateutil.tz import tzical
+from dateutil.tz.tz import _tzicalvtz
+from copyreg import pickle
```

Comment:
```suggestion
import copyreg
```

Reply:
It get's more clear if we use `copyreg.pickle` than `pickle` alone, which can easily mixed up with the python pickle module.

Reply:
abe94fb